src/app/containers/Columns/DndBookmarkColumnCover.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import DndBookmarkColumn from '../../containers/Columns/DndBookmarkColumn'
+import { moveBookmark } from '../../actions'
+
+const mapStateToProps = state => {
+  return { columns: state.bookmarks.columns }
+}
+
+@connect(mapStateToProps)
+export default class DndBookmarkColumnCover extends Component {
```

Comment:
In 90% of the cases when you dont implement an `shouldComponentUpdate`, please use the `React.PureComponent`, this checks the quality of the props and the new props in the render lifecycle to check if he should render itself or not.